<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SieveEditorConfigureServerWidget</class>
 <widget class="QWidget" name="SieveEditorConfigureServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="ServerSieveListWidget" name="serverSieveListWidget"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addServer">
       <property name="text">
        <string>Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifyServer">
       <property name="text">
        <string>Modify...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeServer">
       <property name="text">
        <string>Remove...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>193</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ServerSieveListWidget</class>
   <extends>QListWidget</extends>
   <header>serversievelistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
